src/pages/index.js

```diff
@@ -46,11 +46,13 @@ export async function getStaticProps(context) {
   const targetedHackathon = await fireDb.getTargetedHackathon()
 
   // Uncomment if you want to update config
-  await fireDb.updateConfig(targetedHackathon)
+  // await fireDb.updateConfig(targetedHackathon)
```

Comment:
I don't think so. It seems to me that `updateConfig` is used for batch updating the database fields. I assumed this is probably for the initiation of each website (inserting default fields for each site). Since we are pretty much done developing the cmd-f site, I commented it out to prevent it from overwriting some of the fields.